docs/concepts/mutations.md

```diff
@@ -196,7 +196,7 @@ export const api = createApi({
           body,
         };
       },
-      invalidates: [{ type: 'Posts', id: 'LIST'],
+      invalidates: [{ type: 'Posts', id: 'LIST'}],
```

Comment:
```suggestion
      invalidates: [{ type: 'Posts', id: 'LIST' }],
```

Reply:
oh lol I missed the "commit suggestion" button, but I think I've updated anyway!